tests/trial_tests/test_trial.py

```diff
@@ -181,23 +181,26 @@ def test_check_distribution_suggest_categorical(storage_mode: str) -> None:
 
 @pytest.mark.parametrize("storage_mode", STORAGE_MODES)
 def test_suggest_uniform(storage_mode: str) -> None:
+    low = 0.0
+    high = 3.0
+    x = 1.0
+    y = 2.0
 
-    mock = Mock()
-    mock.side_effect = [1.0, 2.0]
-    sampler = samplers.RandomSampler()
+    relative_search_space = {
+        "x": FloatDistribution(low=low, high=high),
+        "y": FloatDistribution(low=low, high=high),
+    }
+    relative_params = {"x": x, "y": y}
+    sampler = DeterministicRelativeSampler(relative_search_space, relative_params)  # type: ignore
```

Comment:
Not a strong opinion, but how about specifying `low`, `high`, `x`, and `y` directly instead of declaring variables as other test cases do?

https://github.com/optuna/optuna/blob/dab5f6a8ed6124c4390abb8e7373e565ff4b57ce/tests/trial_tests/test_trial.py#L558-L582

Reply:
Ok. I fixed all the tests.